staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -2350,8 +2350,14 @@ func (i *IngressDescriber) Describe(namespace, name string, describerSettings de
 }
 
 func (i *IngressDescriber) describeBackend(ns string, backend *networkingv1beta1.IngressBackend) string {
-	endpoints, _ := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
-	service, _ := i.CoreV1().Services(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	endpoints, err := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Sprintf("<error:%v>", err)
```

Comment:
I'm not sure we're all on the same page. What's the expected behavior if we have a failure to get endpoints or services? is it supposed to print the error and continue, or is it supposed to fail altogether?

As I understand it, this code does the latter (fails altogether), and I would have preferred the former (print the error as part of the description and continue).

Reply:
Thanks for your review @apelisse. The function returns string so it will not fail but print the error with description. See this:
```
for i := range service.Spec.Ports {
```
If `service` is `nil`, it should will never reach this point. That's way I `return fmt.Sprintf("<error:%v>", err)` before.

Reply:
Oh I see, good!